Automattic/pocket-casts-android
# Improve EoY auto scrolling

## Description

Part of feedback from #3169

> The scrolling animation in the first story is not at a constant speed to me, it slows down or speeds up in a random way. Hard to reproduce (checking on my Pixel 7a)

> We should speed up the animation in the second story (number of shows and episodes) a little bit, like a 50% faster

I improved the animation handling by delegating to the framework instead of manually handling the scroll speed.

## Testing Instructions

1. Start PB24.
2. Verify auto-scrolling on Cover, Number of Shows, and Ending stories.

## Screenshots or Screencast 

| Before | After |
| - | - |
| <video src="https://github.com/user-attachments/assets/75de9321-ee06-4196-b9d5-cfa8354b058b" > | <video src="https://github.com/user-attachments/assets/aaeb70ed-6777-455e-9632-143c6108c2e1" > |

## Checklist
